control-plane/tests/test_cli_inspection.py

```
@@ -274,6 +274,15 @@ def __init__(self, server_address: tuple[str, int], handler_class: type) -> None
                         timeout=2,
                     ).read().decode("utf-8")
                 )
+                case_payload = json.loads(
+                    request.urlopen(
+                        (
+                            f"{base_url}/inspect-case-detail"
+                            f"?case_id={promoted_case.case_id}"
+                        ),
+                        timeout=2,
+                    ).read().decode("utf-8")
```

Ruff (0.15.9)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)